config/locales/en.yml

```diff
@@ -862,6 +863,15 @@ en:
             basename: Filename
             delete_originals: Delete Originals
             submit_button: Concatenate
+        deletions:
+          modal:
+            title: File Deletion
```

Comment:
Can we change this modal title to `Delete files` instead

Reply:
Fixed in 62fbc1ba10cbef28ca5e0f5bc9aee03f98487c49